src/me/TomTheDeveloper/InvasionInstance.java

```diff
@@ -423,14 +429,17 @@ public void resetRottenFlesh() {
 
     public void stopGame() {
         if (getPlayersLeft().size() > 0) {
-            getChatManager().broadcastMessage("All-Villagers-Have-Died", ChatColor.RED + "All villagers have died! You lost the game!");
-            getChatManager().broadcastMessage("Reached-Wave-X", "You have reached wave " + ChatColor.AQUA + "%NUMBER%" + ChatColor.GRAY + "!", wave);
-            getChatManager().broadcastMessage("Teleporting-To-Lobby-In-10-Seconds", "You will be teleported to the lobby in " + ChatColor.AQUA + 10 + ChatColor.GRAY + " seconds!");
+        	for(Player p : getPlayers()) {
+                p.sendMessage(LanguageManager.getLanguageFile().get("All-Villagers-Have-Died").toString());
+                p.sendMessage(LanguageManager.getLanguageFile().get("Reached-Wave-X").toString());
+                p.sendMessage(LanguageManager.getLanguageFile().get("Teleporting-To-Lobby-In-10-Seconds").toString());
+            }
         } else {
-            getChatManager().broadcastMessage("All-Players-Have-Died", ChatColor.RED + "All players have died!");
-            getChatManager().broadcastMessage("Reached-Wave-X", "You have reached wave " + ChatColor.AQUA + wave + ChatColor.GRAY + "!", wave);
-            getChatManager().broadcastMessage("Teleporting-To-Lobby-In-10-Seconds", "You will be teleported to the lobby in " + ChatColor.AQUA + 10 + ChatColor.GRAY + " seconds!");
-
+        	for(Player p : getPlayers()) {
+                p.sendMessage(LanguageManager.getLanguageFile().get("All-Players-Have-Died").toString());
+                p.sendMessage(LanguageManager.getLanguageFile().get("Reached-Wave-X").toString());
```

Comment:
Reached Wave X? But you are nowhere getting the wave number. The player will see now in the chat: "You have reached wave %NUMBER%"?